src/libs/API.js

```diff
@@ -400,6 +401,17 @@ function GetAccountStatus(parameters) {
     return Network.post(commandName, parameters);
 }
 
+/**
+ * @param {Object} parameters
+ * @param {String} parameters.debtorEmail
+ * @returns {Promise}
+ */
+function GetIOUReport(parameters) {
+    const commandName = 'GetIOUReport';
+    requireParameters(['debtorEmail'], parameters, commandName);
+    return Network.post(commandName, parameters);
+}
```

Comment:
NAB, curious why this is showing up in this PR since it seems unrelated

Reply:
Ah, I moved them so that theyre in alphabetical order- which seemed to be the convention until recently. Happy to undo this change

Reply:
Ah ok, no worries really. Just seemed out of place in this PR + not sure if we are doing alphabetical order (I promise to definitely mess this up in the near future if we are).

Reply:
ah yeah, I mean maybe if we just keep them inline with where they are in `export {` would be fine, but really anything goes - I end up cmd + f'ing to find them anyway so it doesn't really matter.

Reply:
Alphabetical order was solely for the purpose of alleviating some of my OCD 😁